api/v1/controllers/samples.js

```diff
@@ -197,7 +201,10 @@ module.exports = {
    *  bulk upsert request has been received.
    */
   bulkUpsertSample(req, res/* , next */) {
-    utils.noReadOnlyFieldsInReq(req, helper);
+
+    // make the name post-able
+    const readOnlyFields = helper.readOnlyFields.filter((field) => field !== 'name');
+    utils.noReadOnlyFieldsInReq(req, readOnlyFields);
     const resultObj = { reqStartTime: new Date() };
```

Comment:
@iamigo MIssed this while reviewing the previous PR for readOnlyFields. Do we really want to go through the bulk upsert array before calling the "bulkupsert" function?